tests/test_quadratic_model.py

```diff
@@ -296,6 +296,11 @@ def test_bug982(self):
         self.assertAlmostEqual((i*j).energy({'i': 4294967296, 'j': 4294967296}),
                                1.8446744073709552e+19)
 
+    def test_bug1136(self):
+        # https://github.com/dwavesystems/dimod/issues/1136
+        i = dimod.Integer('i')
+        self.assertEqual((i**2).energy({'i': 1}), 1)
```

Comment:
I think this is better to have with an integer value `!=` 1. If the bug was not fixed, this test would not catch it.

Reply:
Actually, I'm wrong... there would be linear term = 0 and the energy would be 0, sorry